protos/vehicle_data.proto

```diff
@@ -7,170 +7,498 @@ import "google/protobuf/timestamp.proto";
 option go_package = "github.com/teslamotors/fleet-telemetry/protos";
 
 enum Field {
+  // Unknown field, default value
```

Comment:
can you run `make generate-protos`?

Reply:
Done! Sorry for the late response, I just saw your comment.